addon/services/router-scroll.js

```diff
@@ -67,6 +67,16 @@ const RouterScroll = Service.extend({
     }
   },
 
+  get position() {
+    const scrollMap = get(this, 'scrollMap');
+    const stateUuid = get(window, 'history.state.uuid');
+
+    set(this, 'key', stateUuid);
+    const key = getWithDefault(this, 'key', '-1');
+
+    return getWithDefault(scrollMap, key, scrollMap.default);
```

Comment:
Maybe don't use `getWithDefault`?  It only defaults if the value of the key is `undefined`, but if it's `null` then the default value won't be used.
https://github.com/emberjs/ember.js/blob/master/packages/%40ember/-internals/metal/lib/property_get.ts#L192

Reply:
Ah right.  Great point.  Just saw an RFC to deprecate as well.  It is confusing!